### PR TITLE
Enable copy paste HTML for medium editor

### LINKED ISSUE #1688: HTML preformatting not working on Activity feed
**Describe the bug**
On previous version of the BuddyBoss Platform, writing formatted HTML on text editor and pasting on Activity works, after the update, HTML are being stripped down after pasting. This is also happening on forums

**To Reproduce**
1. Go to text editor, preferably in Post or Page where you can create an HTML formatted text in Gutenburg block
2. Create a bulleted list
3. Copy the bulleted list and paste on Activity feed

**Expected behavior**
Tested this on previous version of Platform v1.5.2 and this works, even with the latest theme
After Platform v1.5.3 update, the HTML are being stripped down when pasted on Activity feed, also on forums

**Screenshots**
https://drive.google.com/file/d/1aeIO7evFPdOS-jHTNlYgitDEGSICFXMM/view?usp=sharing

**Support ticket links**
https://secure.helpscout.net/conversation/1314303207/104108
https://secure.helpscout.net/conversation/1324102061/105998

### PR DESCRIPTION
**Pull request by kishanjasani**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:
Disable `cleanPastedHTML` flag in the mediumEditor library. It fixes the copy paste content issue.

Fixes #1688 

### How to test the changes in this Pull Request:

1. Copy the content from editor like Gutenberg
2. Paste that content in the activity editor.
3. See result

### Proof Screenshots or Video
https://www.awesomescreenshot.com/video/5041992?key=fb401b2adfdb871c59b6695801a7fa49

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js [hunk 1]
```diff
@@ -2114,7 +2114,7 @@ window.bp = window.bp || {};
 											cleanTags: [ 'meta', 'div', 'main', 'section', 'article', 'aside', 'button', 'svg', 'canvas', 'figure', 'input', 'textarea', 'select', 'label', 'form', 'table', 'thead', 'tfooter', 'colgroup', 'col', 'tr', 'td', 'th', 'dl', 'dd', 'center', 'caption', 'nav' ],
 											unwrapTags: [ 'ul', 'ol', 'li' ]
 										},
-										imageDragging: false
+										imageDragging: true
```
Comment (chetansatasiya) [inline src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js:2117]: @kishanjasani we not supporting image dropping in editor as we have media uploading.

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
Need changes

File: src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js (hunk 1 above)
Comment (kishanjasani) [reply src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js:2117]: Did changes, As per your Suggestion.

**Review by amincdev — CHANGES_REQUESTED** [review]
@kishanjasani I checked this PR, But the issue is replicable on this PR's branch.
https://www.loom.com/share/09271b14f05a47109028aaeeeade89f2

**Comment by kishanjasani** [comment]
> @kishanjasani I checked this PR, But the issue is replicable on this PR's branch.
> https://www.loom.com/share/09271b14f05a47109028aaeeeade89f2

Yes, @amincdev. It will not work that way. You will have to copy the block. As we are not getting the tag copied properly.

You can refer to this video: https://www.awesomescreenshot.com/video/4858163

**Review by KartikSuthar — APPROVED** [review]

**Comment by KartikSuthar** [comment]
@kishanjasani Code changes look good to me. Merged to dev and pass to QA.

**Comment by KartikSuthar** [comment]
@kishanjasani Please follow the same changes over here:
https://github.com/buddyboss/buddyboss-platform/pull/2699/files#diff-7d4813e9a8ff739edbdc7511ee0da33ed339f543aa50869d4e6f5adcf23481cbL109
https://github.com/buddyboss/buddyboss-platform/pull/2699/files#diff-7d4813e9a8ff739edbdc7511ee0da33ed339f543aa50869d4e6f5adcf23481cbL192

src/bp-templates/bp-nouveau/js/buddypress-messages.jsL835
src/bp-templates/bp-nouveau/js/buddypress-group-messages.jsL881

Also create a same PR for the theme as well.
assets/js/plugins/bbp-reply-ajax.jsL485

**Review by KartikSuthar — DISMISSED** [review]
@sumitbuddyboss @mohit-buddyboss Please text this PR with activity, topic, reply, message, and group message editor.

**Comment by sumitbuddyboss** [comment]
Hello @sourabhmatolia @firozwp 

Need to bump for the next sprint-2. The reason is while testing getting issues and need to check more thoroughly.

Thank you

**Comment by sumitbuddyboss** [comment]
Hello @kishanjasani 

Issue >
If we copy HTML and past in forum new discussion, forum reply, private message, group message then it's not working 

Issue video
https://www.loom.com/share/426e5688a8e743a88f7f153705beceb5

Thank you

**Review by KartikSuthar — DISMISSED** [review]

**Review by amincdev — DISMISSED** [review]

**Review by dipskakadiya — APPROVED** [review]
@kishanjasani Approved Good to merge.

**Comment by JetalDobariya** [comment]
Hello ,

I did verify this PR in qa-approved branch, now Copy - paste HTML code is working fine in Activity feed, Forum Discussion , Replies , Group Message , and Private Message.
Activity Feed : https://www.loom.com/i/a4a439bb9bae415988e6c91b03e7e6c2
Forum Discussion : https://www.loom.com/i/14968ef533bc4a02ba275b611d1fbf2a
Reply Discussion : https://www.loom.com/i/b3adf573faab4b48b65303d753ef9496
Group Message :  https://www.loom.com/i/be2f66998f034e448ff3a02856ae516c
Private Message : https://www.loom.com/i/6f7b4d88e6fb4c958f6c6f4b8030bfd1
All the Update Post : https://www.loom.com/i/135000af931f424f8852ca2a1f5d6c1a

Thanks!

**Pull request merged**